examples/demo.rs

```diff
@@ -78,3 +93,17 @@ fn keyboard_animation_control(
         );
     }
 }
+
+fn update_frame_text(
+    mut q_frame_text: Query<&mut Text, With<FrameText>>,
+    q_animation_player: Query<&AnimationPlayer2D>,
+) {
+    let Ok(mut text) = q_frame_text.get_single_mut() else {
+        return;
+    };
+    let Ok(animation_player) = q_animation_player.get_single() else {
+        return;
+    };
+
+    text.sections[0].value = format!(" current frame: {}", animation_player.frame());
```

Comment:
Leading space.
Also inconsistent with the initial text from line 36 - I think the Should be "current frame: 0"